gatsby-ssr.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const onRenderBody = ({ setPostBodyComponents }) => {
+  const INTERCOM_APP_ID = 'qegbmsy6';
+
+  const intercomMessengerHTML = `
+    window.intercomSettings = {
+      api_base: "https://api-iam.eu.intercom.io",
+      app_id: "${INTERCOM_APP_ID}",
+    };
+
+    (function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',w.intercomSettings);}else{var d=document;var i=function(){i.c(arguments);};i.q=[];i.c=function(args){i.q.push(args);};w.Intercom=i;var l=function(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/${INTERCOM_APP_ID}';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);};if(document.readyState==='complete'){l();}else if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})();
+  `;
+
+  return setPostBodyComponents([
+    <script key="intercom-messenger" dangerouslySetInnerHTML={{ __html: intercomMessengerHTML }} />
+  ]);
+}
+
+export { onRenderBody };
```

Comment:
Curious how this is executed? Don't we need the identity of the user to be signed.. or is that waived where the user is anonymous?

Reply:
I have nothing to sign the identity of the user with 🤷 . It's just an anonymous web visitor.